manual/english/Searching/Search_results.md

```diff
@@ -52,11 +129,98 @@ SELECT id,story_author,comment_author FROM hn_small WHERE story_author='joe' LIM
 +----------------+-------+
 4 rows in set (0.00 sec)
 ```
+
+<!-- request JSON -->
+```JSON
+POST /sql?mode=raw -d "SELECT id,f1,f2 FROM t WHERE f2=2 LIMIT 1; SHOW META"
```

Comment:
Can this be done not via /sql?mode=raw?

Reply:
No, SHOW META requires `mode=raw` in multi-requests as well.